files/image_config/caclmgrd/caclmgrd

```diff
@@ -88,24 +92,48 @@ class ControlPlaneAclManager(object):
     }
 
     def __init__(self):
-        # Open a handle to the Config database
-        self.config_db = ConfigDBConnector()
-        self.config_db.connect()
+        SonicDBConfig.load_sonic_global_db_config()
+        self.config_db_map = {}
+        self.iptables_cmd_ns_prefix = {}
+        self.config_db_map[''] = ConfigDBConnector(use_unix_socket_path=True, namespace='')
+        self.config_db_map[''].connect()
+        self.iptables_cmd_ns_prefix[''] = ""
+        self.docker_bridge_ip = self.get_docker_ip_address(self.iptables_cmd_ns_prefix[''], '')
+        self.namespace_docker_mgmt_ip = {}
+        namespaces = device_info.get_all_namespaces()
+        for front_asic_namespaces in namespaces['front_ns']:
+            self.config_db_map[front_asic_namespaces] = ConfigDBConnector(use_unix_socket_path=True, namespace=front_asic_namespaces)
+            self.config_db_map[front_asic_namespaces].connect()
+            self.iptables_cmd_ns_prefix[front_asic_namespaces] = "ip netns exec " + front_asic_namespaces + " "
+            self.namespace_docker_mgmt_ip[front_asic_namespaces] = self.get_docker_ip_address(self.iptables_cmd_ns_prefix[front_asic_namespaces],
+                                                                                              front_asic_namespaces)
+
+        for back_asic_namespaces in namespaces['back_ns']:
+            self.iptables_cmd_ns_prefix[back_asic_namespaces] = "ip netns exec " + back_asic_namespaces + " "
+            self.namespace_docker_mgmt_ip[back_asic_namespaces] = self.get_docker_ip_address(self.iptables_cmd_ns_prefix[back_asic_namespaces],
+                                                                                             back_asic_namespaces)
+
+    def get_docker_ip_address(self, iptable_ns_cmd_prefix, namespace):
```

Comment:
Suggest renaming function `get_namespace_mgmt_ip()` or `get_namespace_mgmt_ip_addr()` unless you have a reason otherwise.

Reply:
@jleveque Done